Makefile

```diff
@@ -52,41 +51,34 @@ clean:
 	$(BUILD_TOOL) $(XCODEFLAGS) -configuration Release clean
 	$(BUILD_TOOL) $(XCODEFLAGS) -configuration Test clean
 
-install: uninstall package
-	sudo installer -pkg SwiftLint.pkg -target /
+install:
+	swift package clean
```

Comment:
Should this go in `clean`?

Reply:
Before this PR, `clean` was triggered by `install` > `package` > `installables` > `clean`.
I deleted those dependencies considering `make install` executing on Linux.
I got `swift package clean` here to get closer to the previous behavior, but that may not be necessary here. 🤔

Reply:
If we only use `test` as a target that uses `xcodebuild`, we can change it to somewhat simpler one.